src/Tuttifrutti/Cache/Storage/Postgres.hs

```diff
@@ -0,0 +1,110 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Tuttifrutti.Cache.Storage.Postgres where
+
+import           Prelude
+import           RIO.List                             (headMaybe)
+import qualified Tuttifrutti.Cache.Storage            as Storage
+import           Tuttifrutti.Prelude
+
+import qualified Data.Aeson                           as Json
+import           Data.Pool                            (Pool)
+import qualified Data.Pool                            as Pool
+import qualified Data.Range.Range                     as Range
+import qualified Data.Text                            as Text
+import qualified Database.PostgreSQL.Simple           as PG
+import qualified Database.PostgreSQL.Simple.FromField as PG
+import qualified Database.PostgreSQL.Simple.ToField   as PG
+import qualified Database.PostgreSQL.Simple.Types     as PG
+import qualified Tuttifrutti.Persist                  as Persist
+
+newtype TableName = TableName PG.Identifier
+  deriving
+    ( Eq, Ord, Show, IsString)
+
+newPool
+  :: PG.ConnectInfo
+  -> IO (Pool PG.Connection)
+newPool connectInfo =
```

Comment:
That is right, there is no need for that function. I've added it for testing locally, without realizing that we already have a production-grade version of it.

Reply:
I now realized that using `createConnectionPool` directly won't work since it creates the whole persistent `SqlBackend`, and here we just need the `Connection`, no problem, I'll add something to accommodate for that.

Reply:
With this we won't need the pool-creating functions everywhere, but we'll just create pools where they are needed: https://github.com/KSF-Media/tuttifrutti/pull/32